packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md

```diff
@@ -21,36 +19,37 @@ Implement an Okta-hosted user consent dialog box.
 
 ---
 
-## About the user consent dialog box
+## About the user consent dialog
 
-When configured, the Okta-hosted user consent dialog box for OAuth 2.0 or OpenID Connect authentication flows allows users to acknowledge and accept that they are giving an app access to their data. With the correct configuration, Okta displays a consent dialog box that shows which app is asking for access. The dialog box displays the app logo that you specify and also provides details about what data is shared if the user consents.
+When configured, the Okta-hosted user consent dialog for OAuth 2.0 or OpenID Connect authentication flows allows users to acknowledge and accept that they are giving an app access to some of their data. With the correct configuration, Okta displays a consent dialog that shows which app is asking for access to what data. The dialog displays the app logo that you specify and also provides details about what data is shared if the user consents.
```

Comment:
NIT: logo that you specify or logo that you can specify?

Reply:
that you specify. It's describing what a completed dialog looks like.